FreeRTOS/FreeRTOS-Kernel
# merge configRUN_MULTIPLE_PRIORITIES from SMP branch

merge configRUN_MULTIPLE_PRIORITIES from SMP branch

Description
-----------
merge configRUN_MULTIPLE_PRIORITIES from SMP branch

Test Steps
-----------
* RP2040 main full single core and SMP
* WIN32 main full 

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
